TransferList/TransferList.js

```diff
@@ -539,15 +551,17 @@ const TransferListBase = kind({
 			list: 'first',
 			onSelect: setSelected,
 			selectedItems: selectedItems,
-			onSpotlightRight: handleSpotlightRight
+			onSpotlightRight: handleSpotlightRight,
+			showSelection: showSelectionOrder
 		};
 
 		const secondListSpecs = {
 			elements: secondListLocal,
 			list: 'second',
 			onSelect: setSelected,
 			selectedItems: selectedItems,
```

Comment:
i think we can optimize this also
```suggestion
			selectedItems,
```